Zhongli.Services/CommandHelp/CommandHelpDataExtensions.cs

```diff
@@ -59,13 +60,9 @@ private static StringBuilder AppendAliases(this StringBuilder builder, IReadOnly
             if (aliases.Count == 0)
                 return builder;
 
-            builder.AppendLine(Format.Bold("Aliases:"));
-
-            foreach (var alias in FormatUtilities.CollapsePlurals(aliases))
-            {
-                builder.AppendLine($"• {ZhongliConfig.Configuration.Prefix}{alias}");
-            }
-
+            builder.AppendLine(Format.Bold("Aliases:"))
+                .AppendLine($"{FormatUtilities.CollapsePlurals((aliases)).Humanize()}");
```

Comment:
Good approach, probably somewhat hard to read though. Maybe add `Format.Code` like so 
```cs
builder
    .AppendLine(Format.Bold("Aliases:"))
    .AppendLine(FormatUtilities.CollapsePlurals(aliases).Humanize(a => Format.Code(a)));
```

Reply:
Oh and, since `.Humanize()` returns a `string`, no need to string interpolate